rma_sale_analytic/wizards/rma_add_sale.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2023 ForgeFlow S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+
+from odoo import api, models
+
+
+class RmaAddSale(models.TransientModel):
+    _inherit = "rma_add_sale"
+
+    @api.model
+    def _prepare_rma_line_from_sale_order_line(self, line):
+        data = super(RmaAddSale, self)._prepare_rma_line_from_sale_order_line(line)
+        data.update(analytic_account_id=line.analytic_account_id.id)
```

Comment:
Functionally reviewed. It does not work. It does not work in v10. I think the issue in in this line, the analytic account is at order level only. Also this called is never been called. I think this other code must be called instead: https://github.com/ForgeFlow/stock-rma/blob/15.0/rma_sale/models/rma_order_line.py#L168

Reply:
it seems analytic_account_id is not available at the sale order line level.

and with the sale order, it seems working fine

![image](https://github.com/ForgeFlow/stock-rma/assets/94696387/4a46e210-188e-4388-a77e-248eccbaea64)

Reply:
ok, pending to check the other flow. Creating an RMA using this
:
![image](https://github.com/ForgeFlow/stock-rma/assets/19620251/e1b46eef-6fdb-418d-a374-9fda3ac10daa)

Reply:
@AaronHForgeFlow its fixed.

Thanks